### ISSUE TITLE
Code evaluates key rather than check for its presence

### ISSUE THREAD

**Issue by ghost** [stubailo/meteor-reactive-method#4]
On line 28, quoted below:

```
if (cc._reactiveMethodData && cc._reactiveMethodData[serializedArgs]) {
```

`cc._reactiveMethodData[serializedArgs]` should be  `_.has(cc._reactiveMethodData, serializedArgs)` as in:

```
if (cc._reactiveMethodData && _.has(cc._reactiveMethodData, serializedArgs)) {
```

since the former is the _value_ of the key, while the latter is checking for the _presence_ of the key ... which is the condition I believe you were looking for. ;-)

Without this correction, I was getting infinite runs because the method I was calling returns booleans, and the `if` clause was not entered when the return value was `false`. Once I made the correction ... no more infinite runs.

Thanks for this package! It's just what I needed.

**Comment by stubailo** [comment]
Just published 1.0.1 with this fix!

**Comment by ghost** [comment]
Cool!
